src/runmode-napatech.c

```diff
@@ -169,7 +169,10 @@ static void *NapatechConfigParser(const char *device)
     }
 
     /* device+5 is a pointer to the beginning of the stream id after the constant nt portion */
-    conf->stream_id = atoi(device + 2);
+    if (ByteExtractStringUint16(&conf->stream_id, 10, 0, (const char *)(device + 2)) < 0) {
```

Comment:
I suggest changing the comment from `device+5` to `device+2` to reflect the code.

I suggest logging an error if the conversion fails.